### PR TITLE
feat(devx) - Add CTF on testnet

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* reverted c530707bbc071484c2de7e96e95470a2e0312a5f
* Ran the publish.py command and updated the package ids for the CTF challenges

## Type of change

- Documentation Fix

## How the change has been tested

Docs were built localy

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/content/developer/iota-move-ctf/challenge_1.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+import ChallengeVerifier  from '@site/src/components/CTF/ctf-verifier';
+
+
+
+
```
Comment (vivekjain23) [inline docs/content/developer/iota-move-ctf/challenge_1.mdx]: Can we get rid of the extra space used here? WDYT?

File: docs/content/developer/iota-move-ctf/challenge_1.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+This challenge is designed to be an easy introduction, guiding you through the process of interacting with the blockchain, calling functions, and understanding the basics of flag capture.
+
+
+## Deployed Contract Address:
+```
+Package: 0xed1bbeb067ea89b70859641557700997a39bf749b5eedf91e221d56a5d961622
+```
+
+## Contracts
```
Comment (vivekjain23) [inline docs/content/developer/iota-move-ctf/challenge_1.mdx:19]: A line should be left between the two headings.

File: docs/content/developer/iota-move-ctf/challenge_1.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+### `checkin.move`
+```move file=<rootDir>/docs/examples/move/ctf/challenge_1/sources/checkin.move
+```
+
+
```
Comment (vivekjain23) [inline docs/content/developer/iota-move-ctf/challenge_1.mdx:26]:
```suggestion

```

File: docs/content/developer/iota-move-ctf/challenge_2.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+
+## Deployed Contract Addresses:
+```
+Package: 0x2d902cb948748fd2b92ff98e5d0fd46a9ceb4b7501050ef8513b9bc60f515ef8
+Counter: 0xbd21a9287e9f6ea1b69c810cf31e554fa260ff4547b2b94dac3e73239b1c3173
+```
+
+## Contracts
```
Comment (vivekjain23) [inline docs/content/developer/iota-move-ctf/challenge_2.mdx:19]: A line space should be left here as well.

File: docs/content/developer/iota-move-ctf/challenge_2.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+
+## Related Articles
+In this stage of the CTF, you should be familiar with how to use the CLI to call a Move function and pass in the right parameters, as well as a general understanding of the Object Model:
+
+- [IOTA CLI reference](../../references/cli/)
+- [Object Model](../iota-101/objects/object-model.mdx)
+
+
```
Comment (vivekjain23) [inline docs/content/developer/iota-move-ctf/challenge_2.mdx]:
```suggestion
```

**Review by vivekjain23 — CHANGES_REQUESTED** [review]
There are spacing issues in the PR, the functionality looks good though.

File: docs/content/developer/iota-move-ctf/challenge_1.mdx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+## Deployed Contract Address:
+```
+Package: 0xed1bbeb067ea89b70859641557700997a39bf749b5eedf91e221d56a5d961622
+```
+
+## Contracts
+### `checkin.move`
+```move file=<rootDir>/docs/examples/move/ctf/challenge_1/sources/checkin.move
+```
+
```
Comment (vivekjain23) [inline docs/content/developer/iota-move-ctf/challenge_1.mdx:23]:
```suggestion
```

**Review by vivekjain23 — CHANGES_REQUESTED** [review]

**Review by salaheldinsoliman — APPROVED** [review]

File: docs/content/developer/iota-move-ctf/challenge_1.mdx (hunk 1 above)
Comment (lucas-tortora) [reply docs/content/developer/iota-move-ctf/challenge_1.mdx]:
```suggestion
```

File: docs/content/developer/iota-move-ctf/challenge_1.mdx (hunk 2 above)
Comment (lucas-tortora) [reply docs/content/developer/iota-move-ctf/challenge_1.mdx:19]:
```suggestion

## Contracts
```

File: docs/content/developer/iota-move-ctf/challenge_2.mdx (hunk 4 above)
Comment (lucas-tortora) [reply docs/content/developer/iota-move-ctf/challenge_2.mdx:19]:
```suggestion
## Contracts

```

**Review by Ginowine — APPROVED** [review]
Outside leaving 2, 3 and sometime 4 extra spaces before the next line, everything looks great. So I'll approve, but I advise you fix the extra spaces.

**Review by vivekjain23 — APPROVED** [review]

**Pull request merged**